char status
geometry_msgs/Pose cur_pose
float64 cur_time
